client/src/components/DishCard.js

```diff
@@ -20,28 +26,150 @@ const useStyles = makeStyles({
   info:{
     display: 'flex',
     flexDirection:"column",
-    padding:40,
+    paddingLeft:40,
+    paddingTop:40,
+    paddingBottom:40,
+  },
+
+  pplnum:{
+    backgroundColor: "#FF743D",
```

Comment:
We should probably stick to importing the brand color from themes/theme.js

Reply:
I tried that, but did not figure out how to do that, ${brand } doesn't work.

Reply:
Fixed